ush/config_defaults.yaml

```diff
@@ -1777,23 +1767,110 @@ task_run_fcst:
+        jchunk2d: -1
+        ichunk3d: -1
+        jchunk3d: -1
+        kchunk3d: -1
+        itasks: 1
+        quilting: true
+        write_groups: !int '{{ task_run_fcst.WRTCMP_write_groups }}'
+        write_tasks_per_group: !int '{{ task_run_fcst.WRTCMP_write_tasks_per_group }}'
+        num_files: 2
+        filename_base: 'dyn phy'
+        output_file: 'netcdf netcdf'
+        output_fh: '1  -1'
+        nsout: -1
+        output_grid: '{{ task_run_fcst.WRTCMP_output_grid }}'
     namelist:
-      base_file: /path/to/base/input.nml
+      base_file: "{{ user.PARMdir }}/input.nml.FV3"
       update_values:
+        atmos_model_nml:
+          blocksize: !int '{{ task_run_fcst.BLOCKSIZE }}'
+          ccpp_suite: '{{ workflow.CCPP_PHYS_SUITE }}'
         fv_core_nml:
-          k_split: 2
-          n_split: 6
+          target_lon: !int '{{ grid_params.LON_CTR }}'
+          target_lat: !int '{{ grid_params.LAT_CTR }}'
+          nrows_blend: !int '{{ grid_params.HALO_BLEND }}'
+          stretch_fac: !int '{{ grid_params.STRETCH_FAC }}'
+          npx: !int '{{ grid_params.NX + 1 }}'
+          npy: !int '{{ grid_params.NY + 1 }}'
+          layout:
+            - !int '{{ task_run_fcst.LAYOUT_X }}'
+            - !int '{{ task_run_fcst.LAYOUT_Y }}'
+          bc_update_interval: !int '{{ task_get_extrn_lbcs.LBC_SPEC_INTVL_HRS }}'
+        gfs_physics_nml:
+          kice: !int '{{ 9 if workflow.SDF_USES_RUC_LSM else 2 }}'
+          print_diff_pgr: false
+        namsfc:
+          # From sfc_climo_gen
+          fnalbc: '{{ workflow.FIXlam }}/{{ "CRES" | env }}.snowfree_albedo.tileX.nc'
+          fnalbc2: '{{ workflow.FIXlam }}/{{ "CRES" | env }}.facsf.tileX.nc'
+          fntg3c: '{{ workflow.FIXlam }}/{{ "CRES" | env }}.substrate_temperature.tileX.nc'
+          fnvegc: '{{ workflow.FIXlam }}/{{ "CRES" | env }}.vegetation_greenness.tileX.nc'
+          fnvetc: '{{ workflow.FIXlam }}/{{ "CRES" | env }}.vegetation_type.tileX.nc'
+          fnsotc: '{{ workflow.FIXlam }}/{{ "CRES" | env }}.soil_type.tileX.nc'
+          fnvmnc: '{{ workflow.FIXlam }}/{{ "CRES" | env }}.vegetation_greenness.tileX.nc'
+          fnvmxc: '{{ workflow.FIXlam }}/{{ "CRES" | env }}.vegetation_greenness.tileX.nc'
+          fnslpc: '{{ workflow.FIXlam }}/{{ "CRES" | env }}.slope_type.tileX.nc'
+          fnabsc: '{{ workflow.FIXlam }}/{{ "CRES" | env }}.maximum_snow_albedo.tileX.nc'
+          # General fix files
+          fnglac: '{{ workflow.FIXlam }}/global_glacier.2x2.grb'
+          fnmxic: '{{ workflow.FIXlam }}/global_maxice.2x2.grb'
+          fntsfc: '{{ workflow.FIXlam }}/RTGSST.1982.2012.monthly.clim.grb'
+          fnsnoc: '{{ workflow.FIXlam }}/global_snoclim.1.875.grb'
+          fnzorc: '{{ workflow.FIXlam }}/igbp'
+          fnaisc: '{{ workflow.FIXlam }}/CFSR.SEAICE.1982.2012.monthly.clim.grb'
+          fnsmcc: '{{ workflow.FIXlam }}/global_soilmgldas.t126.384.190.grb'
+          fnmskh: '{{ workflow.FIXlam }}/seaice_newland.grb'
```

Comment:
This whole block was derived from entries in the fixed files YAML, but was convoluted and spread over a few places. Condensing it here.